terraform/modules/enclave/prometheus/filebeat.yml.tpl

```diff
@@ -0,0 +1,12 @@
+filebeat.inputs:
+- type: log
+  enabled: true
+  paths:
+    - /var/log/syslog
+
+output.logstash:
+  hosts: ["${logstash_hosts}"]
+  loadbalance: true
+  ssl.enabled: true
+
+tags: ["prometheus", "${environment}"]
```

Comment:
can you update your editor settings to always add newline at end of file please

Reply:
Damn, don't know why I never did that before... That's gonna save me time